src/realm/query_engine.hpp

```diff
@@ -247,13 +247,13 @@ class ParentNode {
 
     virtual std::unique_ptr<ParentNode> clone() const = 0;
 
-    ColKey get_column_key(StringData column_name) const
+    ColKey _get_column_key(StringData column_name) const
     {
         ColKey column_key;
         if (column_name.size() > 0) {
             column_key = m_table.unchecked_ptr()->get_column_key(column_name);
             if (column_key == ColKey()) {
-                throw LogicError(LogicError::column_does_not_exist);
+                // throw LogicError(LogicError::column_does_not_exist);
```

Comment:
I think this needs to get uncommented?

Reply:
The function was not actually used. I just forgot to remove it.